more-examples/main.go

```diff
@@ -0,0 +1,204 @@
+package main
+
+import (
+	"crypto/tls"
+	"crypto/x509"
+	"fmt"
+	"github.com/ravendb/ravendb-go-client"
+	"github.com/ravendb/ravendb-go-client/examples/northwind"
+	"log"
+	"time"
+)
+
+// "Demo" is a Northwind sample database
+// You can browse its content via web interface at
+// http://live-test.ravendb.net/studio/index.html#databases/documents?&database=Demo
+var (
+	dbName        = "TestDapr"
+	serverNodeURL = "http://127.0.0.1:8080"
+
+	secureNodeURL = "https://a.free.nemanja.ravendb.cloud/"
+
+	// if true, we'll show summary of HTTP requests made to the server
+	// and dump full info about failed HTTP requests
+	verboseLogging = true
+
+	// if true, logs all http requests/responses to a file for further inspection
+	// this is for use in tests so the file has a fixed location:
+	// logs/trace_${test_name}_go.txt
+	logAllRequests = false
+
+	// if logAllRequests is true, this is a path of a file where we log
+	// info about all HTTP requests
+	logAllRequestsPath = "http_requests_log.txt"
+)
+
+func getDocumentStore(databaseName string) (*ravendb.DocumentStore, error) {
+	serverNodes := []string{serverNodeURL}
+	store := ravendb.NewDocumentStore(serverNodes, databaseName)
+	if err := store.Initialize(); err != nil {
+		return nil, err
+	}
+	return store, nil
+}
+
+func openSession(databaseName string) (*ravendb.DocumentStore, *ravendb.DocumentSession, error) {
+	store, err := getDocumentStore(dbName)
+	if err != nil {
+		return nil, nil, fmt.Errorf("getDocumentStore() failed with %s", err)
+	}
+
+	session, err := store.OpenSession("")
+	if err != nil {
+		return nil, nil, fmt.Errorf("store.OpenSession() failed with %s", err)
+	}
+	return store, session, nil
+}
+
+func getSecureDocumentStore(databaseName string) (*ravendb.DocumentStore, error) {
+	cerPath := "/Users/nemanjamalocic/projects/RavenDB/ravendb-go-client/more-examples/PEM/free.nemanja.client.certificate.crt"
+	keyPath := "/Users/nemanjamalocic/projects/RavenDB/ravendb-go-client/more-examples/PEM/free.nemanja.client.certificate.key"
```

Comment:
We cannot accept with those sort of constants. Needs to be useful for someone else, and this may cause confusion for users